subprojects/docs/src/docs/userguide/img/java-configurations.dot

```diff
@@ -0,0 +1,21 @@
+digraph configurations {
```

Comment:
We should show `apiElements` as well as the old `compile` and `runtime` configurations here, so people know where the soft-deprecated ones fit in.